dnf-behave-tests/dnf/vars-parsing.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Correct parsing of dnf/vars directory
+ 
+ 
+@bz2091636
+Scenario: Using dnf with non-files in /etc/dnf/vars
+  Given I create directory "/{context.dnf.installroot}/etc/dnf/vars"
```

Comment:
If the path is modified to create a directory in `/etc/dnf/vars` like /{context.dnf.installroot}/etc/dnf/vars/troublemaker" then the step with creating a symlink is not required (directory is also not a file object).

Reply:
It looks like that you forget to push changes